src/Utils/Hooks/useLoadScript.ts

```diff
@@ -23,6 +24,12 @@ export const useLoadScript = ({
     script.id = id
     script.async = true
     script.onload = () => onReady?.()
+    script.onerror = () => {
+      captureException(new Error(`Script load failed: ${id}`), {
+        tags: { source: "script_load_failure" },
+        extra: { scriptId: id, src: script.src },
+      })
+    }
```

Comment:
Nice! I was hoping this would be something global-ish or init-related like this. Be interested to see what your impact will be on the # of users sentry is currently clocking for this.